src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml

```diff
@@ -119,6 +119,7 @@
         <service id="security.access.expression_voter" class="Symfony\Component\Security\Core\Authorization\Voter\ExpressionVoter">
             <argument type="service" id="security.expression_language" />
             <argument type="service" id="security.authentication.trust_resolver" />
+            <argument type="service" id="security.authorization_checker" on-invalid="null" />
```

Comment:
why making it optional ? If you have a voter, you will also have the auth checker (otherwise your voter is useless anyway as it is used by the auth checker)